Source/WebCore/page/scrolling/mac/ScrollerPairMac.mm

```diff
@@ -111,14 +111,23 @@ - (void)scrollerImpPair:(NSScrollerImpPair *)scrollerImpPair setContentAreaNeeds
 
 - (void)scrollerImpPair:(NSScrollerImpPair *)scrollerImpPair updateScrollerStyleForNewRecommendedScrollerStyle:(NSScrollerStyle)newRecommendedScrollerStyle
 {
-    [scrollerImpPair setScrollerStyle:newRecommendedScrollerStyle];
+    UNUSED_PARAM(scrollerImpPair);
+
+    switch (newRecommendedScrollerStyle) {
```

Comment:
We should just write conversion functions so you don't have to switch. And it would be better to refactor so there's only one call to `_scrollerPair->setScrollbarStyle()`.

Reply:
Sounds good! I'll be adding new helpers to go between the two types [per the comment below](https://github.com/WebKit/WebKit/pull/11984#discussion_r1149699758).